### PR TITLE
dx: Dockerfile added

### PR DESCRIPTION
**Pull request by harshit0571**
## Related Issue

- Information about the related issue

Closes: # 

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [x] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/WebXDAO/WebXDAO.github.io/pull/208"></a>

### CODE REVIEW

**Comment by harshit0571** [comment]
Hey, 
Codacy Static Code Analysis  is not approving my pr.
its showing error because i have written WORKDIR in the Dockerfile.
but we have to write WORKDIR in Dockerfile, as it is considered a good practice

**Comment by vinzvinci** [comment]
Anyone remember this issue? I think this happened before and I remember @mkubdev @Panquesito7

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+COPY . .
+
+CMD ["npm","run","dev"]
+
+
+
+# if facing error while building image, paste there two command in terminal.
+
+# export DOCKER_BUILDKIT=0
+#export COMPOSE_DOCKER_CLI_BUILD=0
```
Comment (krshkodes) [inline Dockerfile:15]:
```suggestion
#export COMPOSE_DOCKER_CLI_BUILD=0

```

**Review by krshkodes — CHANGES_REQUESTED** [review]
I think this might be the bug actually

File: Dockerfile (hunk 1 above)
Comment (harshit0571) [reply Dockerfile:15]: they are commented out, i dont think they are causing it

**Comment by harshit0571** [comment]
there is something wrong with github actions file

**Comment by harshit0571** [comment]
it is failing checks

File: Dockerfile (hunk 1 above)
Comment (krshkodes) [reply Dockerfile:15]: not the comment - but the missing endline

Comment (harshit0571) [reply Dockerfile:15]: it is running fine, i tried running it.

Comment (krshkodes) [reply Dockerfile:15]: I am talking for code style/codacy

Comment (harshit0571) [reply Dockerfile:15]: Ohh

Comment (harshit0571) [reply Dockerfile:15]: What should we do now?

**Review by krshkodes — APPROVED** [review]
I think it is all good @harshit0571, won't keep you waiting because of a CI error from our side

**Pull request merged**